README.md

```diff
@@ -120,12 +130,12 @@ try {
 }
 ```
 
-Similarly, you can also process multiple lookups concurrently and await an array of results:
+Similarly, you can also process multiple resolves concurrently and await an array of results:
```

Comment:
I thought since I changed the `lookup` in the code, I thought it would add up to change the describing text above, but it is also okay to use this here. I changed it now @clue